README.md

```diff
@@ -56,22 +56,19 @@ contract YourContract is UsingTellor{
 ```
 
 
-* <b>Miner [Documentation](https://tellor.readthedocs.io/en/latest/MinerSetup/)</b>
+* <b>Miner [Documentation](https://github.com/tellor-io/TellorMiner/)</b>
 
-* <b>General Tellor Developer's [Documentation](https://tellor.readthedocs.io/en/latest/DevDocumentation/)</b>
+* <b>General Tellor Developer's [Documentation](https://app.gitbook.com/@tellor-2/s/tellor-docs/dev-documentation/)</b>
 
 
 ### Useful Links
 
-Metamask - www.metamask.io 
-<br>
+Metamask - www.metamask.io<br>
 Truffle - http://truffleframework.com/
 
 
 #### Maintainers <a name="maintainers"> </a> 
-[@themandalore](https://github.com/themandalore)
-<br>
-[@brendaloya](https://github.com/brendaloya) 
+This repository is maintained by the [Tellor team](https://github.com/orgs/tellor-io/people)
```

Comment:
@themandalore the Org list is now empty, can you please add me, @JGcarv , @brendaloya and yourself there.